feat: outputting literal step text as steps as executed when using verbose go test output (#20)
regen-network/gocuke: README.md

* `Before()`, `After()`, `BeforeStep()`, or and `AfterStep()` can be used to register custom hooks.
* `Tags` and `ShortTags` can be used with tag expressions as described above.
* `NonParallel()` disables parallel tests.

> * `--verbose` or `-v` in your go-tests will emit the current test step definition to stdout while your tests are running, this is useful for testing

Reviewer: ```suggestion
* `--verbose` or `-v` with `go test` will emit the current test step definition to stdout while your tests are running, this is useful for debugging failing tests.
```
